docs/docsite/rst/dev_guide/developing_collections_distributing.rst

```diff
@@ -145,115 +166,116 @@ can set the following in your ``galaxy.yml`` file:
 
-.. _trying_collection_locally:
+  * from the tarball
+  * from your git repository
 
-Trying collections locally
-==========================
+Installing your collection locally from the tarball
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Before you publish your collection, test it out locally. Every time you publish a tarball, you create a :ref:`new version <collection_versions>` of your collection. Testing the collection locally gives you confidence that the new version will contain the functionality you want without unexpected behavior.
+To install your collection locally from the tarball, run ``ansible-galaxy collection install`` and specify the collection tarball. You can optionally specify a location using the ``-p`` flag. For example:
 
-Trying your collection from the tarball
----------------------------------------
+.. code-block:: bash
+
+   collection_dir#> ansible-galaxy collection install my_namespace-my_collection-1.0.0.tar.gz -p ./collections
 
-You can try your collection locally by installing it from the tarball. The following will enable an adjacent playbook to access the collection:
+Install the tarball into a directory configured in :ref:`COLLECTIONS_PATHS` so Ansible can easily find and load the collection. If you do not specify a path value, ``ansible-galaxy collection install`` installs the collection in the first path defined in :ref:`COLLECTIONS_PATHS`.
+
+.. _collections_scm_install:
+
+Installing your collection locally from a git repository
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+To install your collection locally from a git repository, specify the repository and the branch you want to install:
 
 .. code-block:: bash
 
-   ansible-galaxy collection install my_namespace-my_collection-1.0.0.tar.gz -p ./collections
+   collection_dir#> ansible-galaxy collection install git+https://github.com/org/repo.git,devel
 
+.. include:: ../shared_snippets/installing_collections_git_repo.txt
 
-You should use one of the values configured in :ref:`COLLECTIONS_PATHS` for your path. This is also where Ansible itself will
-expect to find collections when attempting to use them. If you don't specify a path value, ``ansible-galaxy collection install``
-installs the collection in the first path defined in :ref:`COLLECTIONS_PATHS`, which by default is ``~/.ansible/collections``.
+Reviewing your collection
+-------------------------
 
-If you want to use a collection directly out of a checked out git repository, see :ref:`hacking_collections`.
+Next, review the collection:
 
-Next, try using the local collection inside a playbook. For examples and more details see :ref:`Using collections <using_collections>`
+* Run a playbook that uses the modules and plugins in your collection. Verify that new features and functionality work as expected. For examples and more details see :ref:`Using collections <using_collections>`. 
+* Check the documentation for typos.
```

Comment:
```suggestion
* Verify the documentation does not contain typos.
```
Clear and more formal.

Reply:
True, but we're not aiming for formality in the upstream docs.